operator/tests/common/delete-securedcluster-errors.yaml

```diff
@@ -372,11 +372,6 @@ metadata:
 ---
 apiVersion: rbac.authorization.k8s.io/v1
 kind: RoleBinding
-metadata:
-  name: sensor-use-scc
```

Comment:
Shouldn't this be renamed to `  name: sensor-use-sccs-for-sensor` rather than removed?

Reply:
I replaced 2 bindings with 1, so one should be removed and the other renamed. The renaming is in this file in line 296.

Note that the `Role` `use-sccs-for-sensor` is mentioned here twice: once for `rbac.authorization.k8s.io/v1` and once for `authorization.openshift.io/v1`

Reply:
I think I see now that the `RoleBinding` should be also twice for both API versions. I will fix that